src/app/shared/models/person.model.ts

```diff
@@ -17,8 +17,8 @@ const Person = t.type({
   // Birthdate: Option(LocalDateTimeFromString),
   // CorrespondenceAddress: Option(t.string),
   DisplayEmail: Option(t.string),
-  // Email: Option(t.string),
-  // Email2: Option(t.string),
+  Email: Option(t.string),
+  Email2: Option(t.string),
```

Comment:
Reicht hier `Option` – sind die leeren Felder immer `null`, nie `undefined`?

Reply:
Ich bin davon ausgegangen, dass alle Emails denselben Typ haben und in den Testdaten scheint es so zu sein. Aber sicher bin ich mir nicht.